app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,83 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if session[:user_id]
+      @user = User.find_by_id(session[:user_id])
```

Comment:
Instead of referencing the session object you can build a `current_user` method and place that in a helpers block in your application controller. 

```ruby
def current_user
  @current_user ||= User.find(session[:id]) if session[:id]
end
```